src/EVEMon.Common/Collections/Global/GlobalCharacterCollection.cs

```diff
@@ -151,7 +152,7 @@ internal void Import(IEnumerable<SerializableSettingsCharacter> serial)
                 // Imports the character
                 SerializableCCPCharacter ccpCharacter = serialCharacter as SerializableCCPCharacter;
                 if (ccpCharacter != null)
-                    this.Add(new CCPCharacter(id, ccpCharacter));
+                    this.Add(new CCPCharacter(id, ccpCharacter), setMonitored: false);
```

Comment:
Could probably get away with not notifying either, as it happens after the for loop

Reply:
At startup nothing is subscribed to OnCharacterCollectionChanged(), so it doesn't make any difference.
It would still make sense to only call it once, at the end of importing all characters.